Code/EnginePlugins/ParticlePlugin/System/ParticleSystemInstance.cpp

```diff
@@ -338,11 +337,10 @@ void ezParticleSystemInstance::ReinitializeStreamProcessors(const ezParticleSyst
 
 ezParticleSystemInstance::ezParticleSystemInstance()
 {
-  m_BoundingVolume = ezBoundingSphere(ezVec3::ZeroVector(), 0.25f);
+  m_BoundingVolume.SetInvalid();
```

Comment:
Was this a problem ?

Reply:
Yes, since the bounding volume is in global space (for non-local space particles) this will be transformed to somewhere in the first frame which could result in particles never spawed for effect that are paused when invisible. It might be not a problem anymore with the new transform and bounds update code in place, but I would leave it in for consistency. The bounds are also invalid if there are no particles spawned yet.